src/world_new/WorldData.cpp

```diff
@@ -23,21 +30,15 @@ WorldData::WorldData(proto::World &protoMsg, rtt::Settings const &settings, std:
     them.reserve(amountThem);
 
     for (auto &each : ours) {
-        auto _ptr = &robots.emplace_back(feedback, each, Team::us);
+        auto _ptr = &robots.emplace_back(feedback, each, Team::us, getBall());
```

Comment:
Why are robots given a ball as well?

Reply:
The robot has a variable "distanceToBall". It needs the ball to actually set this distance. This is how it's also done in the "old" world

Reply:
Fair enough :+1: